doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,109 @@ Here are examples:
+
+   .. prodn::
+      toggle_notation ::= {| @string | @qualid } := @one_term {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | {| @string | @qualid } := @one_term {? ( {+, @toggle_notation_flag } ) }
+      | {| @string | @qualid } {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | {| @string | @qualid } {? ( {+, @toggle_notation_flag } ) }
+      | := @one_term {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | := @one_term {? ( {+, @toggle_notation_flag } ) }
+      | {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      toggle_notation_flag ::= all
+      | only parsing
+      | only printing
+      | in custom @ident
+      | in constr
+
+   Enables or disables notations previously defined with
+   :cmd:`Notation` or :cmd:`Notation (abbreviation)`.
+   This command has no effect on parser modifications made with
+   :cmd:`Reserved Notation`.
+   When multiple
+   informations are provided, the notations enabled or
+   disabled are the ones meeting the conjunction of all constraints.
+
+   This command supports the :attr:`local` and :attr:`global`
+   attributes.
+
+   :n:`@string`
+      Notations to enable or disable. :n:`@string` can be a single
+      token in the notation such as "`->`" or a pattern that matches
+      the notation. See :ref:`locating-notations`.
+
+   :n:`@qualid`
+      :ref:`Abbreviation <Abbreviations>` to enable or disable.
+
+   :n:`@one_term`
+      Enable or disable notations whose interpretation contains :token:`one_term`.
+
+   :n:`all`
+      Enable or disable all notations meeting the given constraints,
+      even if multiple ones. When not provided, there must be a single
+      notation meeting the constraints.
+
+   :n:`only parsing`
+      The notation is enabled or disabled only for parsing.
+
+   :n:`only printing`
+      The notation is enabled or disabled only for printing.
+
+   :n:`in custom @ident`
+      Enable or disable notations in the given :ref:`custom entry
+      <custom-entries>`.
+
+   :n:`in constr`
+      Enable or disable notations in ``constr``. See
+      :ref:`custom entries <custom-entries>`.
+
+   :n:`{? {| : no scope | : @ident } }`
+      If given, only notations in scope :token:`ident` are affected (or
+      :term:`lonely notations <lonely notation>` for :n:`no scope`).
+
+   .. exn:: Unexpected only printing for an only parsing notation.
+
+      Cannot enable or disable for printing a notation that was
+      originally defined as only parsing.
+
+   .. exn:: Unexpected only parsing for an only printing notation.
+
+      Cannot enable or disable for parsing a notation that was
+      originally defined as only printing.
+
+   .. exn:: Found no matching notation to enable.
+
+      No previously defined notation satisfies the given constraints.
+
+   .. exn:: Found no matching notation to disable.
+
+      No previously defined notation satisfies the given constraints.
```

Comment:
These could be one message.  From the command syntax, it will be obvious whether it's enabling or disabling.